### PR TITLE
Added confirmation for `opengb init` command

### LINKED ISSUE #277: Add confirmation for `opengb init` command
**Implementation**

* Say which files were created
* Link to docs on getting started

### PR DESCRIPTION
**Pull request by vr-varad**
**Implementation**

Say which files were created
Link to docs on getting started

**solution Implementation**

The `templateProject` function now returns a Promise of type `string[]`, tracking the paths of created files and directories and resolving with this array. Error handling is implemented to catch and log errors, returning an empty array if creation fails.

Resolves #277

### CODE REVIEW

**Comment by vr-varad** [comment]
Hey @NathanFlurry  I have implemented the solution as discussed. I would appreciate it if you could review it at your earliest convenience. Please let me know if you have any questions or require further clarification. Thank you.

File: src/cli/commands/init.ts [hunk 1]
```diff
@@ -3,10 +3,24 @@ import { GlobalOpts } from "../common.ts";
 import { templateProject } from "../../template/project.ts";
 
 export const initCommand = new Command<GlobalOpts>()
```
Comment (NathanFlurry) [inline src/cli/commands/init.ts:5]: this file has inconsistent indentation, should be using hard tabs

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
welcome, thanks for the pr!

using this in practice, i don't think it actually makes sense to print the files. would simplify intro to something like:

```
Project created successfully

Created config file at backend.yaml

Get started at https://opengb.dev/introduction
```

**Comment by vr-varad** [comment]
@NathanFlurry I have made the required changes.

**Review by NathanFlurry — APPROVED** [review]

**Comment by NathanFlurry** [comment]
@vr-varad thanks for the pr! apologies for the ticket being wrong, i pushed some small tweaks.

**Pull request merged**